testdata/p4_16_errors_outputs/issue2317.p4-stderr

```diff
@@ -1,3 +1,3 @@
-issue2317.p4(16): [--Werror=type-error] error: ==: cannot be applied to action results
+issue2317.p4(16): [--Werror=type-error] error: f == f: cannot be applied to action results
```

Comment:
this one is also confusing

Reply:
This one can be fixed by enhancing `MethodCallExpression`'s `toString()` method. This change would affect ~110 tests so I opted to leave that for a separate PR, but I can make that change here if you prefer.
This is not unique to printing `MethodCallExpression`s used in `==` operations, though. The `MethodCallExpression` will be printed as `f` in other places too.

Reply:
This one can be fixed by enhancing `MethodCallExpression`'s `toString()` method. This change would affect ~110 tests so I opted to leave that for a separate PR, but I can make that change here if you prefer.
This is not unique to printing `MethodCallExpression`s used in `==` operations, though. The `MethodCallExpression` will be printed as `f` in other places too.